index.html

```diff
@@ -27,12 +28,13 @@
       >
         Profile
       </button>
-      <a
-        href="src/online-members/online-members.html"
-        class="online-members-link"
+      <button
+        class="online-members-btn"
+        type="button"
+        onclick="location.href='src/online-members/online-members.html'"
```

Comment:
Why are we adding a button for a link?
Why are we replacing the `<a>` tag with the `<button>` tag?